src/PackagePrivate/AddNormalizedTermsTablesDDL.sql

```diff
@@ -0,0 +1,110 @@
+-- table.
+CREATE TABLE IF NOT EXISTS /*_*/wbt_item_terms (
+  wbit_id                                BIGINT unsigned       NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  wbit_item_id                           INT    unsigned       NOT NULL,
+  wbit_term_in_lang_id                   INT    unsigned       NOT NULL
+) /*$wgDBTableOptions*/;
+
+CREATE INDEX /*i*/wbt_item_terms_item_id ON /*_*/wbt_item_terms (wbit_item_id);
+CREATE UNIQUE INDEX /*i*/wbt_item_terms_term_in_lang_id_item_id ON /*_*/wbt_item_terms (wbit_term_in_lang_id, wbit_item_id);
+
+-- stores a record per term per property per language.
+--
+-- term text, type and language are normalized further through wb_term_in_lang
+-- table.
+CREATE TABLE IF NOT EXISTS /*_*/wbt_property_terms (
+  wbpt_id                                INT unsigned       NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  wbpt_property_id                       INT unsigned       NOT NULL,
+  wbpt_term_in_lang_id                   INT unsigned       NOT NULL
+) /*$wgDBTableOptions*/;
+
+CREATE INDEX /*i*/wbt_property_terms_property_id ON /*_*/wbt_property_terms (wbpt_property_id);
+CREATE UNIQUE INDEX /*i*/wbt_property_terms_term_in_lang_id_property_id ON /*_*/wbt_property_terms (wbpt_term_in_lang_id, wbpt_property_id);
+
+-- stores a record per term per text per language.
+--
+-- term text and language are normalized further through wb_text_in_lang table.
+CREATE TABLE IF NOT EXISTS /*_*/wbt_term_in_lang (
+  wbtl_id                          INT unsigned       NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  wbtl_type_id                     INT unsigned       NOT NULL,
+  wbtl_text_in_lang_id             INT unsigned       NOT NULL
+) /*$wgDBTableOptions*/;
+
+CREATE INDEX /*i*/wbt_term_in_lang_type_id_text_in ON /*_*/wbt_term_in_lang (wbtl_type_id);
+CREATE UNIQUE INDEX /*i*/wbt_term_in_lang_text_in_lang_id_lang_id ON /*_*/wbt_term_in_lang (wbtl_text_in_lang_id, wbtl_type_id);
+
+-- stores a record per term text per language.
+--
+-- text is normalized through wb_term_text table.
+CREATE TABLE IF NOT EXISTS /*_*/wbt_text_in_lang (
+  wbxl_id                         INT unsigned       NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  wbxl_language                   VARBINARY(10)      NOT NULL,
+  wbxl_text_id                    INT unsigned       NOT NULL
+) /*$wgDBTableOptions*/;
+
+CREATE INDEX /*i*/wbt_text_in_lang_language ON /*_*/wbt_text_in_lang (wbxl_language);
+CREATE UNIQUE INDEX /*i*/wbt_text_in_lang_text_id_text_id ON /*_*/wbt_text_in_lang (wbxl_text_id, wbxl_language);
+
+-- stores a record per text value that are used in different terms
+-- in different languages.
+CREATE TABLE IF NOT EXISTS /*_*/wbt_text (
+  wbx_id                             INT unsigned         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  wbx_text                           VARBINARY(255)       NOT NULL
+) /*$wgDBTableOptions*/;
+
+CREATE UNIQUE INDEX /*i*/wbt_text_text ON /*_*/wbt_text (wbx_text);
```

Comment:
This file is an unchanged copy of what is in Wikibase.git master, except for the above index, which was renamed from `wbt_text`. The test caught that the index had the same name as the table.